### PR TITLE
2nd batch REUSE compliance

### PR DESCRIPTION
**Pull request by tobiasd**
Half way through...

adds REUSE compliant information to

* src
* view
* mods
* static
* tests

directories

### CODE REVIEW

File: view/php/default.php [hunk 1]
```diff
@@ -1,3 +1,17 @@
...
+ * SPDX-FileCopyrightText: 2010-2024 the Friendica project
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ * The site template for pure content (e.g. (modals)
+ *
+ * This template is used e.g for bs modals. So outputs
+ * only the pure content
+ */
+
+*>
```
Comment (annando) [inline view/php/default.php]: That doesn't look right.

File: view/php/minimal.php [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+ * SPDX-FileCopyrightText: 2010-2024 the Friendica project
+ *
+ * SPDX-License-Identifier: AGPL-3.0-or-later
+ * The site template for pure content (e.g. (modals)
+ *
+ * This template is used e.g for bs modals. So outputs
+ * only the pure content
+ */
+
+*>
```
Comment (annando) [inline view/php/minimal.php]: Looks wrong as well.

Comment (tobiasd) [reply view/php/minimal.php]: added a blank line

File: view/php/default.php (hunk 1 above)
Comment (tobiasd) [reply view/php/default.php]: added a blank line

**Pull request merged**